racetime_bot/bot.py

```diff
@@ -28,7 +26,7 @@ class Bot:
 
     continue_on = [
         # Exception types that will not cause the bot to shut down.
-        websockets.ConnectionClosed,
+        # websockets.ConnectionClosed,
```

Comment:
I need to figure out what should be here for asyncio websockets

Reply:
@andrew12 Are you intending to resolve this?

Reply:
does aiohttp raise an exception on a clean exit?

Reply:
Hi from the future, do you guys know any way I could test what happens for a clean disconnect? I think it might just be [https://docs.aiohttp.org/en/v0.18.2/api.html#aiohttp.errors.DisconnectedError](DisconnectedError) or some subclass but not sure.

Reply:
Would probably make sense to continue for all HTTP issues, clean or otherwise Since they (probably) don't point to any issue with the bot itself.

Reply:
<img width="514" alt="Screen Shot 2021-11-15 at 2 40 48 PM" src="https://user-images.githubusercontent.com/170843/141843673-98bce1a7-0d15-410b-a156-22fa940dc879.png">

Looks like I could just use ClientError? Or ClientResponseError and ClientConnectionError

Reply:
I changed it to ClientError for now. I want to test this then will make this not a draft